source/fundamentals/builders/sort.txt

```diff
@@ -71,86 +58,91 @@ sample collection, ``sort_example``, that contains the following documents:
    {"_id": 2, "letter": "a", "food": "donuts and coffee"}
    {"_id": 6, "letter": "c", "food": "maple donut"}
 
+The Sorts Class
+---------------
 
-Sorting Direction
------------------
+The ``Sorts`` class is a builder that provides static factory methods for all sort
+criteria operators supported by MongoDB. These methods return a ``Bson`` object
+that you can pass to the  ``sort()`` method of a ``FindIterable`` instance or to 
+``Aggregates.sort()``. If you want to learn more about ``Aggregates``, see our 
+:doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
 
-The ``Sorts`` class provides methods for specifying the direction of your sort.
-The direction of your sort can either be **ascending** or **descending**.
-An ascending sort orders your results from smallest to largest. A
-descending sort orders your results from largest to smallest.
+The following two code snippets are equivalent, however we recommend you use the
+``Sorts`` builder to specify your sort criteria:
 
-Here are some examples of data sorted in ascending order:
+.. code-block:: java
 
-* Numbers: 1, 2, 3, 43, 43, 55, 120
-* Dates: 1990-03-10, 1995-01-01, 2005-10-30, 2005-12-21 
-* Words (ASCII): Banana, Dill, carrot, cucumber, hummus
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   //Recommended
```

Comment:
Purely personal preference:
```suggestion
   // Recommended
```

Reply:
Thank you for the note. This is the style we have been using as well. I think I need to update my Intellij settings to automatically lead code comments with spaces.